microsoft-edge/webview2/release-notes.md

```diff
@@ -93,6 +104,132 @@ SDKs/packages are Prerelease or Release
+   
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2File](/microsoft-edge/webview2/reference/win32/icorewebview2file?view=webview2-1.0.1774.30&preserve-view=true)
+   * [ICoreWebView2File::get_Path](/microsoft-edge/webview2/reference/win32/icorewebview2file?view=webview2-1.0.1774.30&preserve-view=true#get_path)
+
+* [ICoreWebView2ObjectCollectionView](/microsoft-edge/webview2/reference/win32/icorewebview2objectcollectionview?view=webview2-1.0.1774.30&preserve-view=true)
+   * [ICoreWebView2ObjectCollectionView::get_Count](/microsoft-edge/webview2/reference/win32/icorewebview2objectcollectionview?view=webview2-1.0.1774.30&preserve-view=true#get_count)
+   * [ICoreWebView2ObjectCollectionView::GetValueAtIndex](/microsoft-edge/webview2/reference/win32/icorewebview2objectcollectionview?view=webview2-1.0.1774.30&preserve-view=true#getvalueatindex)
+
+* [ICoreWebView2WebMessageReceivedEventArgs2](/microsoft-edge/webview2/reference/win32/icorewebview2webmessagereceivedeventargs2?view=webview2-1.0.1774.30&preserve-view=true)
+   * [ICoreWebView2WebMessageReceivedEventArgs2::get_AdditionalObjects](/microsoft-edge/webview2/reference/win32/icorewebview2webmessagereceivedeventargs2?view=webview2-1.0.1774.30&preserve-view=true#get_additionalobjects)
+
+---
+
+
+<!-- ------------------------------ -->
+* The Profile Cookie Manager API supports profile management.  The `CookieManager` property enables the host app to get the cookie manager for the profile.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2Profile`
+   * [CoreWebView2Profile.CookieManager property](/dotnet/api/microsoft.web.webview2.core.corewebview2profile.cookiemanager?view=webview2-dotnet-1.0.1774.30&preserve-view=true#microsoft-web-webview2-core-corewebview2profile-cookiemanager)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2Profile`
+   * [CoreWebView2Profile.CookieManager property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1774.30&preserve-view=true#cookiemanager)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Profile5](/microsoft-edge/webview2/reference/win32/icorewebview2profile5?view=webview2-1.0.1774.30&preserve-view=true)
+   * [ICoreWebView2Profile5::get_CookieManager](/microsoft-edge/webview2/reference/win32/icorewebview2profile5?view=webview2-1.0.1774.30&preserve-view=true#get_cookiemanager)
+
+---
+
+
+<!-- ====================================================================== -->
+## 1.0.1829-prerelease
+
+Release Date: May 8, 2023
+
+[NuGet package for WebView2 SDK 1.0.1829-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1829-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 115.0.1829.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### General
+
+

```

Comment:
Add Promotions section?  

```
###### Promotions

The following APIs have been promoted to Stable and are now included in this Release SDK.
```

not seeing strict alternation in Search POV, so not sure if the Promotions section is always present:
![image](https://user-images.githubusercontent.com/45407486/236360190-396927dc-899b-428f-a196-aa9c80be2bf1.png)

Reply:
Question: Are there Promotions sections that are empty, in previous sections?  
Answer: No; per a recent normalization pass, any Promotions sections that exist have content and lead-in, either:
The following APIs have been promoted to Stable and are now included in this Release SDK.
The following APIs have been promoted from Experimental to Stable in this Prerelease SDK.

There are 57 #### General   headings/sections
There are 39 ###### Promotions   headings/sections

The policy/design is evidently:  omit Promotions if there are no promotions; don't do:
```
###### Promotions
There are no Promotions in this release|prerelease.
```

Reply:
Seems we don't have experimental features, only pre-release stable this time. I have adjusted the heading accordingly.

Reply:
@novac42 @peiche-jessica 
i removed hidden comment
```
<!-- Seems we don't have experimental features, only pre-release stable this time.-->
```
and added standard string:
```
The following APIs have been promoted from Experimental to Stable in this Prerelease SDK.
```